src/test/scala/zio/sqs/ZioSqsSpec.scala

```diff
@@ -0,0 +1,201 @@
+package zio.sqs
+
+import java.net.URI
+
+import org.elasticmq.rest.sqs.SQSRestServerBuilder
+import org.scalatest.{ FlatSpec, Matchers }
+import scalaz.zio.{ DefaultRuntime, Task }
+import software.amazon.awssdk.auth.credentials.{ AwsBasicCredentials, StaticCredentialsProvider }
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.Message
+
+class ZioSqsSpec extends FlatSpec with Matchers {
+  "ZioSqsSpec" should "send a message" in {
+    val result = for {
+      server <- Task(
```

Comment:
As we discussed together, it would be great to use ZManaged to handle the clean acquisition/release of the client, and also it can be re-used in every test so that there's no duplication.